### PR TITLE
[WIP]Validation feedback mobile doc changes

### PR DESCRIPTION
**Pull request by gkassabli**
#### Description
This PR addresses feedback from Validation in VAL-241

#### Tests
Run docs locally and ensure that feedback has been addressed

### CODE REVIEW

File: docs/content/mobile/android/local-deploy.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Before reading this document, make sure you are familiar with:
 
   * [Setting up Android Support for the GDK]({{urlRoot}}/content/mobile/android/setup)
-  * [Ways to test your Android client]({{urlRoot}}/content/mobile/android/ways-to-test)
+  * [Ways to test your Android client]({{urlRoot}}/content/mobile/android/ways-to-run)
```
Comment (jessicafalk) [inline docs/content/mobile/android/local-deploy.md]: also need to change the name of the link

File: docs/content/mobile/android/ways-to-run.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+# Ways to run your Android client
 
 Before reading this document, make sure you are familiar with:
 
   * [The SpatialOS GDK for Unity]({{urlRoot}}/content/intro-reference)
   * [Mobile support overview]({{urlRoot}}/content/mobile/overview)
   * [Setting up iOS support for the GDK]({{urlRoot}}/content/mobile/android/setup)
 
-Unity provides multiple ways to test your Android [client-worker]({{urlRoot}}/content/glossary#client-worker). We integrated them all to work with [SpatialOS]({{urlRoot}}/content/glossary#spatialos-runtime). This documentation describes the benefits of the different options.
+Unity provides multiple ways to run your Android [client-worker]({{urlRoot}}/content/glossary#client-worker). We integrated them all to work with [SpatialOS]({{urlRoot}}/content/glossary#spatialos-runtime). This documentation describes the benefits of the different options.
```
Comment (jessicafalk) [inline docs/content/mobile/android/ways-to-run.md]: we need to rephrase that, let's chat about that later

File: docs/content/mobile/android/ways-to-run.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
 ## In the Editor
-For standard workflows and for minor changes, run your game in the Editor. When the build platform is set for Android, it executes code that is in preprocessor `#if UNITY_ANDROID` clauses. This way, you have the full capabilities and ease of use of the Unity Editor, while still testing code that would otherwise only run on a mobile device.
+For standard workflows and for minor changes, run your game in the Editor. When the build platform is set for Android, it executes code that is in preprocessor `#if UNITY_ANDROID` clauses. This way, you have the full capabilities and ease of use of the Unity Editor, while still running code that would otherwise only run on a mobile device.
```
Comment (jessicafalk) [inline docs/content/mobile/android/ways-to-run.md]: we actually should also mention asmdef...

**Comment by gkassabli** [comment]
Yeah, I definitely want to wait with this change until validation is done, just wanted to start the work on addressing the issues in it

**Comment by jessicafalk** [comment]
should we close this PR as ollie is doing the changes in his PR?

**Comment by gkassabli** [comment]
Closing in favor of https://github.com/spatialos/gdk-for-unity/pull/723

**Pull request closed without merging**